Project-Euler/Project-Euler/Program.cs

```diff
@@ -8,8 +8,31 @@ namespace Project_Euler
 {
     class Program
     {
-        static void Main(string[] args)
+        static void Main()
         {
+            PrintIntro();
+            PrintMenuOptions();//Eventually this method will not be called in main directly. This is for testing purposes.
+        }
+
+        public static void PrintIntro()
+        {
+            Console.WriteLine("\nWelcome To My Project Euler Repository!");
+            //Add a couple more lines, maybe?
+        }
+
+        public static void PrintMenuOptions() //This method will be private once I write the method that will call it. blah blah
+        {
+            Console.WriteLine("\n\n\tSelect One Of The Following Options:\n" +
+                              "\n\tA - See All Problems and Solutions" +
+                              "\n\tB - See One Specific Problem and Solution" +
+                              "\n\tQ - Quit");
+
+            Console.ReadLine();
```

Comment:
You shouldn't be reading input in a method called PrintMenu, I would create another method which gets the user input